### ISSUE TITLE
Some code issues

### ISSUE THREAD

**Issue by codingidea** [lyakaap/VAT-pytorch#12]
1. In VAT-pytorch/vat.py/line7-16, running_mean and running_var in BatchNorm layers will keep updating if you only set track_running_stats to False. Setting the layer into eval mode would be better;

2. VAT-pytorch/train_CIFAR10.py/line53-58, both labeled and unlabeled samples are used to compute the regularization
loss in the original paper , and they are in different batch size(64, 128 respectively).
